### PR TITLE
Add flags for filtering xpack audit events

### PR DESCRIPTION
**Pull request by vinn946**

### CODE REVIEW

File: run.sh
Comment (marcinc) [inline run.sh]: index events include comes with default set - so this could be potentially skipped / or included only when specified. Add extra options above to the README.

File: run.sh
Comment (marcinc) [inline run.sh]: The 2 options above should be `XPACK_AUDIT_LOGFILE_EVENTS_{EXCLUDE/INCLUDE}`?

**Review by marcinc — CHANGES_REQUESTED** [review]

File: README.md
Comment (marcinc) [inline README.md]: Worth giving an example of how multiple fields can be excluded i.e. comma separated list of values

File: config/elasticsearch.yml
Comment (marcinc) [inline config/elasticsearch.yml]: Does it expect an array or single string with comma separated event names? If the former then you'd have to ensure values are interpolated as expected so it doesn't break configuration.

**Review by KashifSaadat — APPROVED** [review]
LGTM

File: README.md
Comment (marcinc) [inline README.md]: Minor but maybe rename to include `XPACK_SECURITY_...`. Same below.

**Review by marcinc — CHANGES_REQUESTED** [review]

**Review by marcinc — APPROVED** [review]
LGTM, assuming it worked fine in test.

**Comment by marcinc** [comment]
Just don't forget to squash before merging :)

**Pull request merged**
